user-registration-app/src/app/animations.ts

```diff
@@ -1,19 +1,18 @@
-import { animate, animateChild, group, query, style, transition, trigger } from "@angular/animations";
+import { animate, animateChild, group, query, style, transition, trigger } from '@angular/animations';
```

Comment:
All double quotes have been moved to single quotes to match the rest of the codebase